src/cookieSyncWithConsent.js

```diff
@@ -168,7 +173,11 @@ function parseQueryParam(name, urlSearch) {
  * Otherwise it will return a default value
  */
 function sanitizeEndpoint(value) {
```

Comment:
can we DRY this up as a util? duplicate here: https://github.com/freestarcapital/prebid-universal-creative/pull/2/files#diff-a0d5cdd49fa319d677864e6a26d89fe9b79e8cf6c7ce340c915e9521438c9d6cR173